doc/source/examples/multiphysics/warming-up-a-viscous-fluid/warming-up-a-viscous-fluid.rst

```diff
@@ -322,6 +322,64 @@ After the fluid has been heated up by the right plate, the temperature is really
     :alt: Rescaled domain with temperature (t = 7)
     :width: 30%
 
+
+Horizontal domain
+~~~~~~~~~~~~~~~~~
+
+Several adjustments have to be made in the `.prm` to turn the domain clockwise, so that it becomes horizontal, with the upper wall being the no slip and thermal insulation boundary condition, and the lower wall with the flow in the y-direction (:math:`v=2`) and heating at Tw:
+
+* in ``subsection mesh``: ``set grid arguments = 0, 0 : 1, 0.5 : true``
+* in ``subsection analytical solution``, ``subsection temperature``: 
+   ``set Function expression = Tw+(((rho*nu)*v*v)/(2*K))*(1-(y/B)*(y/B))``
+* and most importantly, the ``boundary conditions`` subsection should state explicitly ``outlet`` for the fluid boundary conditions, as the ``number`` of boundary conditions should be adapted to use the bottom and top wall (see the `deal.II documentation on hyper_rectangle grid generator <https://www.dealii.org/current/doxygen/deal.II/namespaceGridGenerator.html#a56019d263ae45708302d5d7599f0d458>`_ for further details):
+
+.. code-block:: text
+
+	subsection boundary conditions
+	  set number                  = 4
+	    subsection bc 0
+	    set id = 0
+		set type              = outlet
+	    end
+	    subsection bc 1
+	    set id = 1
+		set type              = outlet
+	    end
```

Comment:
I'm not sure I understand, here you don't have only outlets no? one is an inlet and one is an outlet? Be careful  because the outlet boundary condition penalizes inflow, it's not like a nothing boundary condition, it's different depending on the value of beta and it's not meant to be used as an inlet

Reply:
You can specify a nothing BC, you just need to specify the id of the bc.
For example, if you a nothing BC on the id 0, you can have something like.
subsection bc 0
 set id =1
 set type ...
end
subsection bc 1
 set id = 2
 set type ...
end

It is not mandatory that the bc "number" matches the boundary id.  In all cases, it would be better to make the horizontal example either with a nothing bc or without specifying an outlet (specifying nothing) because this "inlet" behavior of the outlet is not desired, it's because the default beta value is small.

Reply:
Oh ok I got it! Sorry, you are right, I will change that right away